tests/testthat/test-pstr.R

```diff
@@ -92,25 +92,25 @@ test_that("thresholds yield expected low, medium, and high risk categories", {
     type = "ipr",
   )
 
-  default_low_mid <- formals(pstr)$low_threshold
+  default_low_mid <- 1/3
```

Comment:
I make this simpler.

We could have made the previous approach work with `eval()` 

``` r
devtools::load_all()
#> ℹ Loading tiltIndicator

formals(pstr)$low_threshold
#> 1/3

class(formals(pstr)$low_threshold)
#> [1] "call"
# Bad
formals(pstr)$low_threshold + 1
#> Error in formals(pstr)$low_threshold + 1: non-numeric argument to binary operator

class(eval(formals(pstr)$low_threshold))
#> [1] "numeric"
# Good
eval(formals(pstr)$low_threshold) + 1
#> [1] 1.333333
```

<sup>Created on 2023-05-16 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>